scripts/exgfs_wave_post_pnt.sh

```diff
@@ -250,7 +250,7 @@
     ymdh=$(${NDATE} -"${WAVHINDH}" "${PDY}${cyc}")
     YMD=${ymdh:0:8}
     tstart="${ymdh:0:8} ${ymdh:8:2}0000"
-    N=$(( ($FHMAX_WAV_PNT - $FHMIN_WAV) * 3600 / $DTPNT_WAV + 1 ))
+    N=$(( (FHMAX_WAV_PNT - FHMIN_WAV) * 3600 / DTPNT_WAV + 1 ))
```

Comment:
I think this might be accidentally undoing a change from your ufswm PR?

Reply:
this was a recent update required by Dave just now.